pylint/lint/run.py

```diff
@@ -184,7 +184,13 @@ def __init__(
                 file=sys.stderr,
             )
             sys.exit(32)
-        if linter.config.jobs > 1 or linter.config.jobs == 0:
+        if linter.config.jobs >= 0:
+            if self._plugins:
+                warnings.warn(
+                    "Running pylint in parallel with custom plugins is not currently supported.",
+                    UserWarning,
+                )
+                sys.exit(32)
```

Comment:
Hm Ok I'm not super convinced anymore after realizing that if you run
`pylint test.py --enable=all --enable-all-extensions` 
on any test.py module raises this warning now...
which is prob because of `--enable-all-extensions`.  

Maybe the code needs to be that config.jobs can be either `0` or anything >1? that would make more sense...